### PR TITLE
Remove public scripting APIs that are not ready for release

### PR DESCRIPTION
**Pull request by tmat**

### CODE REVIEW

**Comment by tmat** [comment]
test eta please

**Comment by jaredpar** [comment]
The ETA failure appears to be real.

**Comment by tmat** [comment]
test eta please

**Comment by tmat** [comment]
@jaredpar I know, I've been changing ETA to fix the issues.

File: src/VisualStudio/InteractiveServices/Interactive/ResetInteractive.cs [hunk 1]
```diff
@@ -204,7 +204,9 @@ private static string GetReferenceString(Reference reference)
                 return null;
             }
 
-            // TODO: This shouldn't directly depend on GAC, rather we should have some kind of "reference simplifier".
+            return reference.Path;
+
+#if TODO // TODO: This shouldn't directly depend on GAC, rather we should have some kind of "reference simplifier".
```
Comment (jaredpar) [inline src/VisualStudio/InteractiveServices/Interactive/ResetInteractive.cs]: Do we have a bug for this?

**Pull request merged**